tests/monotouch-test/Security/CertificateTest.cs

```diff
@@ -534,9 +534,18 @@ public void MailRaw ()
 		}
 
 		[Test]
+#if MONOMAC
+		[Ignore ("For some reason cert.Handle is coming back as IntPtr.Zero in test.  Works fine in a normal mac app.")]
```

Comment:
iirc the full managed implementation returns `IntPtr.Zero` but the one that use Apple TLS does not. system mono or mobile profile ?